uber/templates/registration/form.html

```diff
@@ -146,6 +146,27 @@
         </div>
     </div>
 
+    <div class="form-group">
```

Comment:
We want only people who have access to Accounts to see this. We'll need a `{% if c.HAS_ACCOUNTS_ACCESS %}` if-check in the template here.